tests/Acoustics.Test/AnalysisPrograms/Recognizers/GenericRecognizerTests.cs

```diff
@@ -310,10 +310,9 @@ public void TestWhistleAlgorithm()
 
             var results = recognizer.Recognize(recording, config, 100.Seconds(), null, this.TestOutputDirectory, null);
 
-            // add next two lines because cannot find spectrogram included with the test results.
             // Used for debugging only
-            var image = SpectrogramTools.GetSonogramPlusCharts(results.Sonogram, results.NewEvents, results.Plots, null);
-            image.SaveAsPng("C:/temp/image.png");
+            var image1 = SpectrogramTools.GetSonogramPlusCharts(results.Sonogram, results.NewEvents, results.Plots, null);
+            image1.Save(this.TestOutputDirectory + "\\image1.png");
```

Comment:
You cannot use backslashes - our tests run on non-windows paltforms.

Use the `this.TestOutputDirectory.CombineFile("WhistleSpectrogram.png")`